MTM-58098 release notes for Date-Filtering-doesnt-work-in-Audit-Logs-… (#2174)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-10.20.419.0_Date-Filtering-doesnt-work-in-Audit-Logs-for-Alarms.md

---

date:
title: Audit logs for repeating alarms are created with the date of the last alarm update
product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-OG_650_b2
    label: Core platform
build_artifact: 
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-58098
version: 10.20.419.0
---

> To create audit log for repeating alarms, the alarm creation date was used, which resulted in audit logs having past dates. This issue has been fixed and now audit logs for repeating alarms are created with the date of the last alarm update.

Reviewer: ```suggestion
Audit logs for repeating alarms used the alarm creation date which resulted in audit logs having past dates. This issue has been fixed and now audit logs for repeating alarms are created with the date of the last alarm update.
```

---

> date: 2024-04-26

Reviewer 2: ```suggestion
date: 
```

Reviewer 2: The date refers to the deployment date on Zone 1 and will be added at a later time

Author: Changed.

> title: Create an audit log with the date of the last alarm update

Reviewer 2: ```suggestion
title: Audit logs for repeating alarms are created with the date of the last alarm update
```

Author: Changed.

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-OG_650_b2
    label: Core platform
build_artifact: 
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-58098
version: 10.20.419.0
---

> To create an audit for a repeating alarm, the alarm creation date was used, which resulted in the audit logs having past dates.

Reviewer 2: ```suggestion
To create audit log for repeating alarms, the alarm creation date was used, which resulted in audit logs having past dates. This issue has been fixed and now audit logs for repeating alarms are created with the date of the last alarm update.
```

Author: Changed.
